### ISSUE TITLE
Viresclient requires markupsafe==2.0.1

### ISSUE THREAD

**Issue by mfbenitezp** [ESA-VirES/VirES-Python-Client#75]
In a recent implementation of VirES using poetry and miniconda, there was an issue that requires downgrading markupsafe to 2.0.1 once I did this, it fixes the issue on my side.

**Comment by smithara** [comment]
Thank you for the report

I'm not familiar with poetry (and what issues you might have mixing that with conda) - but can you try replacing the `viresclient` specified with a dev branch on github. Using pip this is done with:
```
pip install git+https://github.com/ESA-VirES/VirES-Python-Client@allow-jinja2v3
```
I guess it is something similar with poetry. That should let it install without complaint with higher versions of markupsafe (actually, with the direct dependency of Jinja2). I think the functionality of viresclient should still work unaffected - let me know if not!

**Comment by smithara** [comment]
@mfbenitezp I released a new version (0.10.1) with which you should be able to remove the pin of markupsafe. The problem was that viresclient 0.10.0 was holding back the upgrade of jinja2, which itself was required in order to fix compatibility with a newer version of markupsafe. (https://github.com/pallets/markupsafe/issues/286)
